tests/util/kubeUtils.go

```diff
@@ -77,12 +78,10 @@ func DeleteNamespace(n string) error {
 
 // NamespaceDeleted check if a kubernete namespace is deleted
 func NamespaceDeleted(n string) (bool, error) {
-	output, err := Shell("kubectl get namespace %s", n)
+	output, err := ShellSilent("kubectl get namespace %s -o name", n)
 	if strings.Contains(output, "NotFound") {
-		log.Infof("namespace %s deleted\n", n)
 		return true, nil
 	}
-	log.Infof("namespace %s not deleted yet\n", n)
```

Comment:
those logs were ok no ?

Reply:
The calling function also logged the same information.

```bash
    if !namespaceDeleted {
        log.Errorf("Failed to delete namespace %s after %v seconds", k.Namespace, maxAttempts)
        return nil
    }

    log.Infof("Namespace %s deletion status: %v", k.Namespace, namespaceDeleted)
```